base/deprecated.jl

```diff
@@ -1269,11 +1269,13 @@ end
 # (2) base/linalg/qr.jl
 # (3) base/linalg/lq.jl
 
-@deprecate find(x::Number)            find(!iszero, x)
-@deprecate findnext(A, v, i::Integer) findnext(equalto(v), A, i)
-@deprecate findfirst(A, v)            findfirst(equalto(v), A)
-@deprecate findprev(A, v, i::Integer) findprev(equalto(v), A, i)
-@deprecate findlast(A, v)             findlast(equalto(v), A)
+@deprecate find(x::Number)             find(!iszero, x)
+@deprecate findnext(A, v, i)           findnext(equalto(v), A, i)
```

Comment:
You should probably keep `::Integer` for deprecations, else they are called incorrectly and make tests fail.

Reply:
Yeah, that was what I was worried about. I'm abandoning the idea of providing a sentinel argument, so keeping this `::Integer` isn't so bad anymore.

Reply:
Note that these are just deprecations, we don't need to support new features for them anyway.

Reply:
I was worried about deprecations causing ambiguities.